hook/scriptwrapper_test.go

```diff
@@ -110,11 +104,11 @@ func TestHookScriptsAreGeneratedCorrectlyOnWindowsBatch(t *testing.T) {
 	// See: https://pkg.go.dev/fmt > ctrl-f for "%%"
 	scriptTemplate := `@echo off
 SETLOCAL ENABLEDELAYEDEXPANSION
-SET > "%s"
+buildkite-agent env > "%s"
```

Comment:
Checking the exact contents of the wrapper script in a test is a change detector, and there's a test for the functionality of the wrapper above. Is there any value beyond testing the file was written and is nonempty?

Reply:
yeah, i get where you're coming from here. these tests were originally added as regression tests when we replaced [this nonsense](https://github.com/buildkite/agent/blob/345e259bc18bdd62cc86407acb75d9a17aa77415/hook/scriptwrapper.go#L117-L141) with the current template solution. i think there's a little bit of value in making double sure that we don't accidentally mess with the templates, but i agree that the value isn't huge.